compass-model/src/test/java/de/dfki/asr/compass/model/PrefabSetDeepCopyTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of COMPASS. It is subject to the license terms in
+ * the LICENSE file found in the top-level directory of this distribution.
+ * (Also available at http://www.apache.org/licenses/LICENSE-2.0.txt)
+ * You may not use this file except in compliance with the License.
+ */
+package de.dfki.asr.compass.model;
+
+import java.io.IOException;
+import org.testng.annotations.Test;
+import static org.testng.Assert.*;
+
+public class PrefabSetDeepCopyTest {
+
+	@Test
+	public void deepCopyPrefabSetShouldCreateCopy() throws IOException, ClassNotFoundException {
+		PrefabSet originalSet = initializePrefabSet();
+		PrefabSet clone = (PrefabSet) originalSet.deepCopy();
+		assertEquals(originalSet.getName(), clone.getName());
+	}
+
+	@Test
+	public void deepCopyPrefabSetShouldCopyChildren() throws IOException, ClassNotFoundException {
+		Boolean childrenCopied = true;
+		PrefabSet originalSet = initializePrefabSet();
+		PrefabSet clone = (PrefabSet) originalSet.deepCopy();
+		if (originalSet.getChildren().size() != clone.getChildren().size()) {
+			childrenCopied = false;
+		}
+		for (PrefabSet child: originalSet.getChildren()) {
+			int childIndex = originalSet.getChildren().indexOf(child);
+			if (!(clone.getChildren().get(childIndex).getName().equals(child.getName()))) {
+				childrenCopied = false;
+			}
+		}
+		assertTrue(childrenCopied);
```

Comment:
My gut feeling is that this test would look much less convoluted if there were multiple asserts. Or is short-circuiting a problem here?
I might just try refactoring it to check.